### ISSUE TITLE
How to install and run ANSYS Geometry Service/PygGeometry on Linux

### ISSUE THREAD

**Issue by c-eriksson86** [ansys/pyansys-geometry#1452]
### Description of the modifications

Hey,
As I have understood it, pyGeometry uses ANSYS Geometry Service. ANSYS Geometry Service is available as a product flag when installing ansys silent. However, it is not supported on linux, i.e. the flag doesn't work there. 
So, how to install ANSYS Geometry Service and PyGeometry on Linux?
Am I missing something simple here?

**Comment by RobPasMue** [comment]
Hi @c-eriksson86 - the Ansys Geometry Service is currently not available on Linux, despite the development team is working to make it available for Linux (i.e. it's part of the roadmap).

Closing issue!

**Comment by c-eriksson86** [comment]
Hey,
Thanks for the quick reply!

**Comment by RobPasMue** [comment]
Of course, anytime! I must say though, the library itself (PyAnsys Geometry) will have the same API for all the backends. Meaning that if you develop your script against a backend like the Windows version of the Ansys Geometry Service, once the Linux version is available it should work out of the box (no changes needed) =) Hope this helps!

Kind regards!

**Comment by kmahajan-cadfem** [comment]
@RobPasMue, is this information correct for newest version as well?

according to the [documentation](https://geometry.docs.pyansys.com/version/stable/getting_started/local/index.html#launch-a-local-session), it is possible to launch the linux local geometry service.


we are looking for  2 things

1. unified installer for linux which packs geometry service with it, as per [documentation with flag](https://geometry.docs.pyansys.com/version/dev/getting_started/faq.html#how-is-the-ansys-geometry-service-installed)
2. launch the geometry service locally without the containers

**Comment by kmahajan-cadfem** [comment]
sorry missed the last line while copy pasting

`GeometryExitedError: Geometry service connection terminated: Service.Start() has not been called.`

**Comment by RobPasMue** [comment]
Seems like a licensing issue -- do you have the license environment variables defined?

**Comment by kmahajan-cadfem** [comment]
which env variable should be set, I have tried following 
-  `ANSYSLIC_DIR`
-  `ANSYLMD_FILE`
- `LICENSE_SERVER`

<img width="536" height="41" alt="Image" src="https://github.com/user-attachments/assets/48ea5e79-29a7-49f2-8d35-748fd22ee1dc" />

<img width="405" height="20" alt="Image" src="https://github.com/user-attachments/assets/a61fa7ea-cbe1-46af-8a12-de2db357a2fb" />

**Comment by RobPasMue** [comment]
> * `ANSYSLMD_LICENSE_FILE`
> * `LICENSE_SERVER`

These two should work yeah. Can we see the server logs on your side? You can use https://geometry.docs.pyansys.com/version/stable/api/ansys/geometry/core/modeler/Modeler.html#Modeler.get_service_logs AFAIK

**Comment by kmahajan-cadfem** [comment]
@RobPasMue, anything else that I can try, to get it working?
is it still related to the licensing?

**Comment by RobPasMue** [comment]
Maybe @b-matteo or @RyanJWard can help on this since they have troubleshooted licensing issues in the past

**Comment by RyanJWard** [comment]
@kmahajan-cadfem What Linux distro are you using? You can try setting ANSYSCL252_DIR = /opt/ansys/v252/GeometryService/licensingclient

**Comment by kmahajan-cadfem** [comment]
I am using the `Rocky Linux 8.10`

**Comment by RyanJWard** [comment]
@kmahajan-cadfem @RobPasMue There is a current limitation where the service won't run on Rocky 8. Not sure if we should open a new issue to track this as a known limitation

**Comment by kmahajan-cadfem** [comment]
Thank you very much for the information

could you please let me  know which distributions are tested and works without problems for the Geometry Service?

it would be great to have this information documented somewhere for future
